apps/judicial-system/digital-mailbox-api/src/app/modules/cases/case.service.ts

```diff
@@ -95,6 +96,16 @@ export class CaseService {
     lang?: string,
   ): Promise<CaseResponse> {
     const response = await this.fetchCase(id, nationalId)
+    const defendant = response.defendants[0]
+
+    if (!defendant) {
+      throw new NotFoundException('No defendant found for this case')
+    }
+
+    if (defendant.subpoenas && defendant.subpoenas[0].subpoenaId) {
```

Comment:
Weren't we only going to perform this request if we don't already have a service status for the subpoena?

Reply:
Thanks! 🙏